benchmarks/multiply_dimensionwise/Project.toml

```diff
@@ -0,0 +1,10 @@
+[deps]
```

Comment:
Since we can always re-generate benchmark data, we can also just the binary files from this PR. Can I just `git rm` them or do I need some special git magic?

Reply:
Yes, you need special git magic or a new branch. The problem is that if you remove these files and merge the branch later to `master`, the commits leading up to the merge are preserved in history and so are the (now deleted files). You could either
* rewrite your branch's commit history as if the files were never added, or
* create a new branch that contains all changes/additions from this branch *except* the files (by manually copying/applying diffs)
The first one would generally be the nicer solution (since it preserves all relevant history), but also means more work (see also [here](https://trixi-framework.github.io/Trixi.jl/stable/github-git/#Remove-large-file-from-repository)).

Reply:
I would definitely like to keep the history, so the second approach doesn't seem feasible. Since the binary files are so small in this case (smaller than something like `src/solvers/dg/2d/dg.jl`), I would like to just `git rm` them now. I'll promise to never push binaries again... I fear that we can really get into trouble when modifying the history badly, i.e. concerning the Julia registry etc.

Reply:
OK, sounds good enough ;-)